resources/tag_models/event.json

```diff
@@ -0,0 +1,29 @@
+{
+  "tag": {
+	"api_key": "", // String - API key - Make sure the request is from a valid client (idea nicked from Keen.io) useful if a page gets copied onto a Russian website and creates noise
+	"version": "", // Version of the tracking client e.g. "JS v1.2"
+	"type": "event", // Type of request (page or event)
+	"id": "", // ID of this request, must be unique
+	"offset": 0 // Delay of this tag between event happening and being sent to server - milliseconds
+  },
+
+  "event": {
+	"page_id": "", // ID of the page on which this event happened.
+	"parent_id": "", // Parent event ID if this is a child event.
```

Comment:
There are two threads here - one is our opinion on whether a hierarchical data model for analytics makes sense.  Alberto and I don't think it does, but if there's no flexibility on that or everyone else is in favour of heirarchy then fine.  In any case, we are willing to stop engaging with this point if it's proving unhelpful.

The other thread is the API used by developers.  As long as we do not have to maintain data between scopes that should have no need to communicate with one another, that will mitigate our strongest issue with the linking between events.  So to be clear, that means that no event triggered within a component will ever need to reference any event triggered outside of that component.